Taskfile.yml

```
@@ -74,3 +74,25 @@ tasks:
       - cp ./docs/en/releases/template.ipynb ./docs/en/releases/jijmodeling-{{.CLI_ARGS}}.ipynb
       - 'yq -i ''.parts[-1].chapters |= [{"file": "releases/jijmodeling-{{.CLI_ARGS}}"}] + .'' docs/ja/_toc.yml'
       - 'yq -i ''.parts[-1].chapters |= [{"file": "releases/jijmodeling-{{.CLI_ARGS}}"}] + .'' docs/en/_toc.yml'
+
+  sync_paired_notebooks:
+    desc: Synchronize paired notebooks and markdown files
+    cmds:
+      - find docs \( -name "*.ipynb" -o -name "*.md" \) ! -name "template.ipynb" -exec uv run jupytext --sync {} \;
+
+  create_paired_notebooks:
+    desc: Create paired notebooks and markdown files
+    cmds:
+      - find docs \( -name "*.ipynb" -o -name "*.md" \) ! -name "template.ipynb" -exec uv run jupytext --set-formats ipynb,md:myst {} \;
+
+  check_paired_notebooks:
+    desc: Check the consistency of paired notebooks and markdown files
+    cmds:
+      - |
+        LOG=$(find docs \( -name "*.ipynb" -o -name "*.md" \) ! -name "template.ipynb" \
```

上と同様、`*.md` のみマッチで良いと思います。
```suggestion
        LOG=$(find docs -o -name "*.md" \
```